docs/src/pages/guides/guide-react/index.md

```diff
@@ -24,6 +24,8 @@ If you're using [Create React App](https://create-react-app.dev/) (or a fork of
 npx -p @storybook/cli sb init --type react_scripts
 ```
 
+If you're using [Create React App](https://create-react-app.dev/) (or a fork of `react-scripts`) with TypeScript, you shold follow this [instructions](https://gist.github.com/shilman/bc9cbedb2a7efb5ec6710337cbd20c0c).
+
```

Comment:
@shilman This is a change on master.. Should we merge this but then remove from next, since in 6.0.0 this will be 0-config?